Device/build.sh

```diff
@@ -115,7 +155,25 @@ stop_build_container() {
     docker rm "$SETUP_CONTAINER_ID"
 }
 
+write_wifi_credentials() {
```

Comment:
Why don't you use 'sed' to just replace the user and password?

Reply:
I guess it's just a matter of preference. If the file doesn't exist, we're safe with that too.

But it got me thinking, what if the user does not want to specify WiFi credentials and wants to login manually?
Is it enough to just reset the chip again? 
If not, we would need to remove "all" the code that contains the wifi setup. (?)